build/Defaults/Desktop/app.config

```diff
@@ -3,7 +3,6 @@
 <!-- The default app.config for test projects that do not specify one -->
 <configuration>
   <appSettings>
-    <add key="xunit.appDomain" value="denied"/>
```

Comment:
Why strip the denial of AppDomains? IIRC, we had this to ensure Desktop tests were not dependent on AppDomains, so (optimally) no changes would be needed to get them running on .netcore